flow/project.py

```diff
@@ -3161,13 +3244,19 @@ def _register_operations(self):
                 'pre': pre_conditions.get(func, None),
                 'post': post_conditions.get(func, None)}
 
+            # Update operation hooks
+            self._operation_hooks[name].update(Hooks.from_dict(self.hook[func]))
```

Comment:
If I remember correctly, that's by design. I always considered hooks to be registered on a *per-instance* level, and not to be inherited. That's because hooks are something that executed for a particular runtime and unlike the DAG, not something that is *inherent* to a project's workflow.